tooling/cli/templates/mobile/android/app/build.gradle.kts

```diff
@@ -8,6 +9,15 @@ plugins {
     id("{{this}}"){{/each}}
 }
 
+val keystoreProperties = Properties()
+try {
+  // release builds require the file to be set up
+  keystoreProperties.load(FileInputStream(rootProject.file("keystore.properties")))
+} catch (e: Exception) {
+  // load a dummy file for debug purposes (signing config won't be used)
+  keystoreProperties.load(FileInputStream(rootProject.file("keystore.properties.dummy")))
+}
```

Comment:
maybe we should just include `keystore.properties` by default instead of the dummy one, or even just remove this altogether and just document how users might want to add signing later on if they wish.

Reply:
i'd be fine with this on docs instead @FabianLars @simonhyll

Reply:
It used to be in the docs but was lost in recent refactors of tauri documentation and whether we merge this PR or not, I think we will still need to have it documented